server/api/region.go

```diff
@@ -333,7 +401,12 @@ func (h *regionsHandler) GetRegions(w http.ResponseWriter, r *http.Request) {
 	rc := getCluster(r)
 	regions := rc.GetRegions()
 	regionsInfo := convertToAPIRegions(regions)
-	h.rd.JSON(w, http.StatusOK, regionsInfo)
+	b, err := regionsInfo.marshal(r.Context())
```

Comment:
how about cache this result in a few seconds to decrease memory alloc?

Reply:
In oncall it is called by some version of lightning, the interval between their calls is greater than a few seconds, I'm not sure if it has any requirements for immediacy?

Reply:
In this pr, we will solve "after the connection is disconnected, GetRegions will continue to be processed and occupy a large amount of memory until it is completely handled before the memory is released" firstly. If need to add cache for it, we can do it in another pr.